runtime/v2/runc/pause/sandbox.go

```diff
@@ -71,6 +73,17 @@ func (p *pauseService) StartSandbox(ctx context.Context, req *api.StartSandboxRe
 	return &api.StartSandboxResponse{}, nil
 }
 
+func (p *pauseService) Platform(ctx context.Context, req *api.PlatformRequest) (*api.PlatformResponse, error) {
+	log.Debugf("platform request: %+v", req)
+
+	platform := types.Platform{
+		OS:           runtime.GOOS,
```

Comment:
Side note: the user might use other runtime which is compatible with runC commandline.
The platform info might be incorrect. We can add some information to notice user about this.